src/app/pages/server-details/server-details.component.ts

```diff
@@ -762,6 +773,24 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
       return undefined;
     }
 
+    if(name === 'storages') {
+      let html = '<ul>';
+      (prop as any[]).forEach((s: any, index: number) => {
+        html += `<li>${s.size}GB ${s.storage_type ? s.storage_type : ''}</li>`;
+      });
+      html += '</ul>';
+      return html;
+    }
+
+    if(name === 'gpus') {
+      let html = '<ul>';
+      (prop as any[]).forEach((s: any, index: number) => {
+        html += `<li>${s.model}${s.manufacturer ? (' (' + s.manufacturer + ')') : ''}${s.family ? ' family: ' + s.family + ',' : ''}${s.memory ? ' memory: ' + s.memory + 'MB' : ''}</li>`;
```

Comment:
If we are formatting this nicely, then I'd construct this as manufacturer (if available), then family (if available), then the exact model (if available) -- see how it's done at the FAQ specs already:

![image](https://github.com/user-attachments/assets/07cfee8d-018d-4e3d-9a8a-bdf606d29049)

Then show the following fields in parentheses, but note the need for the space between the value and unit (like in all cases everywhere in the app):

1. memory amount
2. `Firmware version` if know
3. `BIOS version` if known
4. `Clock rate` (filled in by `graphics_clock`) if available